Makefile

```diff
@@ -0,0 +1,35 @@
+# @todo use this APP_CONTAINER in all commands instead of opendorme_app_1
+APP_CONTAINER=$(docker ps | grep php-fpm | awk '{print $1}')
+DC = docker exec -it opendorme_app_1 /bin/bash -c
+
+connect-app: ## Connect to app container
+	docker exec -it opendorme_app_1 /bin/bash
+
+composer-install: ## Install composer dependencies
+	$(DC) 'composer install'
+
+migrate: ## Migrate database
+	$(DC) 'php artisan migrate'
+
+seed: ## Migrate database and seed
+	$(DC) 'php artisan migrate --seed'
+
+test: ## Run tests
+	$(DC) 'php artisan test'
+
```

Comment:
Could we add a command here doing running horizon?
`$(DC) 'php artisan horizon'`

Reply:
DONE